contrib/core-contract-tests/tests/pox-4/signers-voting.prop.test.ts

```diff
@@ -0,0 +1,137 @@
+import fc from "fast-check";
+import { it, expect } from "vitest";
+import { Cl } from "@stacks/transactions";
+
+it("should return correct reward-cycle-to-burn-height", () => {
+  fc.assert(
+    fc.property(
+      fc.constantFrom(...simnet.getAccounts().values()),
+      fc.nat(),
+      (account: string, reward_cycle: number) => {
+        // Arrange
+        const { result: pox_4_info } = simnet.callReadOnlyFn(
+          "pox-4",
+          "get-pox-info",
+          [],
+          account,
+        );
+        const first_burnchain_block_height =
+          // @ts-ignore
+          pox_4_info.value.data[
+            "first-burnchain-block-height"];
+        const reward_cycle_length =
+          // @ts-ignore
+          pox_4_info.value.data[
+            "reward-cycle-length"];
+
+        // Act
+        const { result: actual } = simnet.callReadOnlyFn(
+          "signers-voting",
+          "reward-cycle-to-burn-height",
```

Comment:
Compared with the [existing tests in `signers-voting.test.ts`](https://github.com/stacks-network/stacks-core/blob/059ae88deae783081abdf159debcd2bd8d62e200/contrib/core-contract-tests/tests/pox-4/signers-voting.test.ts#L25-L35) file as well:

![image](https://github.com/stacks-network/stacks-core/assets/287532/7f70c1c9-11ba-4bf8-81ac-c90f4b800dbd)